packages/oui-navbar/src/brand/navbar-brand.html

```diff
@@ -1,7 +1,8 @@
 <a class="oui-navbar__brand"
     ng-attr-aria-label="{{::!!$ctrl.ariaLabel ? $ctrl.ariaLabel : null}}"
     ng-attr-title="{{::!!$ctrl.ariaLabel ? $ctrl.ariaLabel : null}}"
-    ng-href="{{::$ctrl.href}}">
+    ng-href="{{::$ctrl.href}}"
+    ng-click="$event.preventDefault(); $ctrl.onNavigateLink()">
```

Comment:
Why can't we have a simple `ng-click="$ctrl.onClick()"`, like we have here https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-navbar/src/link/navbar-link.html#L13 ?

To avoid having functional part in the component, the `preventDefault` should be in the callback function in the Manager.